src/ng/directive/ngModel.js

```diff
@@ -878,6 +878,141 @@ NgModelController.prototype = {
    */
   $overrideModelOptions: function(options) {
     this.$options = this.$options.createChild(options);
+  },
+
+  /**
+   * @ngdoc method
+   *
+   * @name  ngModel.NgModelController#$processModelValue
+
+   * @description
+   *
+   * Runs the model -> view pipeline on the current
+   * {@link ngModel.NgModelController#$modelValue $modelValue}
+   *
+   * The following actions are performed by this method:
+   *
+   * - the $modelValue is run through the {@link ngModel.NgModelController#$formatters $formatters}
+   * and the result is set to the {@link ngModel.NgModelController#$viewValue $viewValue}
+   * - the `ng-empty` or `ng-not-empty` class is set on the element
+   * - if the $viewValue has changed
+   *   - {@link ngModel.NgModelController#$render $render} is called on the control
+   *   - the {@link ngModel.NgModelController#$validators $validators} are run and
+   *   the validation status is set.
+   *
+   * This method is called by ngModel internally when the bound scope value changes.
+   * Application developers usually do not have to call this function themselves.
+   *
+   * This function can be used when the $viewValue or the rendered DOM value of the control should
+   * be updated after a user input.
+   *
+   * For example, consider a text input with an autocomplete list (for fruit, where the items are
+   * objects with a name and an id.
+   * A user enters `ap` and then selects `Apricot` from the list.
+   * Based on this, the autocomplete widget will call $setViewValue({name: 'Apricot', id: 443}),
```

Comment:
It is better to wrap code in `\. (E.g. `$setViewValue(...)` here or `ctrl.$processModelValue()` and `$viewValue` below.)

Reply:
Apparently if you want to escape \`, you need to place the `\` _before_, not _after_ the \` 😇